.github/workflows/ci.yml

```
@@ -0,0 +1,310 @@
+        run: "poetry run invoke unittest"
+  publish_gh:
+    needs:
+      - "unittest"
+    name: "Publish to GitHub"
+    runs-on: "ubuntu-22.04"
+    if: "startsWith(github.ref, 'refs/tags/v')"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v4"
+      - name: "Set up Python"
+        uses: "actions/setup-python@v4"
```

actionlint (1.7.11)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)